### PR TITLE
add caption style to the dropdown items

### PR DESCRIPTION
**Pull request by eonwarped**

### CODE REVIEW

File: src/views/my-items/MyItemsPage.tsx [hunk 1]
```diff
@@ -267,25 +267,31 @@ export default function MyItemsPage() {
           <StyledMenuBar>
             <StyledMenuItem>
               {isMobile ? <SortedIcon /> : <p>{t('sorted_by')}</p>}
-              <Dropdown
-                selected={sortedBy.text}
-                options={sortedByOptions.map(({ text }) => text)}
-                onSelect={text => setSortedBy(sortedByOptions.find(o => o.text === text) || sortedByOptions[0])}
-              />
+              <Caption>
```
Comment (0xthk) [inline src/views/my-items/MyItemsPage.tsx:270]: We should put `<Caption>` in Dropdown.tsx if we want to style dropdown component.

**Review by 0xthk — CHANGES_REQUESTED** [review]

File: src/views/my-items/MyItemsPage.tsx (hunk 1 above)
Comment (eonwarped) [reply src/views/my-items/MyItemsPage.tsx:270]: Wasn't sure if we wanted all dropdowns to have it. but I can do that too and see how it looks where used

Comment (eonwarped) [reply src/views/my-items/MyItemsPage.tsx:270]: @Appppo the Dropdown already has 'SmallContent' on mobile, do we just want to make that even smaller? (14px -> 12px)

**Comment by pyc92** [comment]
all dropdown menu will be replaced soon, no need to spend time on these component now.

**Pull request closed without merging**